src/components/generic/Scatter.tsx

```diff
@@ -1,16 +1,17 @@
 import * as React from 'react';
 import { GenericComponent, IGenericProps, IGenericState } from './GenericComponent';
-import Card from '../Card';
-import { ScatterChart, Scatter as ScatterLine, XAxis, YAxis, ZAxis, CartesianGrid, Tooltip, Legend, ResponsiveContainer }  from 'recharts';
-
-import * as _ from 'lodash';
 import * as moment from 'moment';
-
+import * as _ from 'lodash';
+import { ScatterChart, Scatter as ScatterLine, XAxis, YAxis, ZAxis, CartesianGrid, Tooltip, Legend, ResponsiveContainer } from 'recharts';
+import Card from '../Card';
 import colors from '../colors';
 var { ThemeColors } = colors;
 
 interface IScatterProps extends IGenericProps {
-  theme?: string[]
+  theme?: string[],
+  xKey?: string,
+  yKey?: string,
+  zKey?: string,
 }
```

Comment:
The `static defaultProps` does not match this interface, I'd change the interface to march `xDataKey` format